src/components/layout/nav/ToolbarMenu.tsx

```diff
@@ -65,60 +65,32 @@ export const useActiveNaveItem = () => {
 };
 
 const ToolbarMenu: React.FC = () => {
-  const activeItem = useActiveNaveItem();
+  const activeItem = useActiveNavItem();
   const isMobile = useIsMobile();
 
-  const navItemSx = useCallback(
-    (selected: boolean): SxProps<Theme> => {
-      return isMobile
-        ? {
-            py: 1,
-            px: 3,
-            textDecoration: 'none',
-            textOverflow: 'ellipsis',
-            overflowX: 'hidden',
-            whiteSpace: 'nowrap',
-            display: 'block',
-            color: selected ? 'primary.main' : 'text.primary',
-            fontWeight: selected ? 600 : 400,
-            '&.Mui-focusVisible': {
-              outlineOffset: '-2px',
-            },
-            backgroundColor: selected
-              ? (theme: Theme) => lighten(theme.palette.primary.light, 0.9)
-              : undefined,
-            border: '2px solid transparent',
-          }
-        : {
-            fontWeight: 600,
-            fontSize: 14,
-            px: { xs: 0.5, lg: 2 },
-            color: 'text.primary',
-            backgroundColor: selected
-              ? (theme: Theme) => alpha(theme.palette.primary.light, 0.12)
-              : undefined,
-          };
-    },
-    [isMobile]
-  );
-
   return TOOLBAR_MENU_ITEMS.map((item) => {
     let navItem = isMobile ? (
-      <MenuItem
-        component={RouterLink}
-        to={item.path}
-        sx={navItemSx(activeItem === item.activeItemPathIncludes)}
-        data-testid={item.id}
+      <MobileMenuItem
         key={item.id}
-      >
-        {item.title}
-      </MenuItem>
+        title={item.title as string}
+        selected={activeItem === item.activeItemPathIncludes}
+        path={item.path}
+      />
     ) : (
       <ButtonLink
```

Comment:
so since this is rendered within a `List` component, it looks like you end up with `<ul><a>item</a</ul>` - which isn't valid. We could take out the ul as a quick fix. Although ideally the markup would be more like `<nav><ul><li><a>item</a</li></ul></nav>` for accessibility

Reply:
Good point, do you think it's ok to add <li> as a quick fix as I've done here? I agree we should add `nav` too, but when I briefly tried it messed with your styling. We do have an a11y ticket for that somewhere so maybe better to revisit and fix when we have a clearer idea of the long term vision for this nav.

Reply:
Looks good, thanks for the quick fix!